ember-accounts/app/mixins/analytics.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+
+//A temporary solution to get GA shipped, Really what I want here
+//is conventionally queued Analytics events - @discorick
+export default Ember.Mixin.create({
+  actions: {
+    purchase(model) {
+      window.analytics.page({
+        url: `/settings/${this.get('model.login')}/upgrade`,
```

Comment:
I thought about it, honestly I omitted it to match all the other payloads to keep things looking consistent.